book/07-git-tools/1-git-tools.asc

```diff
@@ -1,10 +1,10 @@
 [[_git_tools]]
-== Git Tools
+== Mga Git na Kasangkapan
 
-By now, you’ve learned most of the day-to-day commands and workflows that you need to manage or maintain a Git repository for your source code control.
-You’ve accomplished the basic tasks of tracking and committing files, and you’ve harnessed the power of the staging area and lightweight topic branching and merging.
+Sa ngayon, natutunan mo ang karamihan ng araw-araw sa mga utos at mga workflow na kailangan mong pamahalaan o pangalagaan ang Git na repositoryo para sa iyong source code na kontrol.
+Nagawa mo na ang mga pangunahing gawain ng pagsubaybay at pag-commit ng mga file, at na-harness mo ang kapayarihan sa stanging na lawak at magaan na paksa na pag-branch at pag-merge.
```

Comment:
- translate `na-harness` to `napakinabangan`
- `stanging` to `staging`

Reply:
done change